specification/guestconfiguration/GuestConfiguration.Management/models.tsp

```diff
@@ -0,0 +1,801 @@
+   */
+  @visibility(Lifecycle.Read)
+  configurationModeFrequencyMins?: float32 = 15;
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+model AssignmentReport {
+  /**
+   * ARM resource id of the report for the guest configuration assignment.
+   */
+  @visibility(Lifecycle.Read)
+  id?: string;
+
+  /**
+   * GUID that identifies the guest configuration assignment report under a subscription, resource group.
+   */
+  @visibility(Lifecycle.Read)
+  reportId?: string;
+
+  /**
+   * Configuration details of the guest configuration assignment.
+   */
+  assignment?: AssignmentInfo;
+
+  /**
+   * Information about the VM.
+   */
+  vm?: VMInfo;
+
+  /**
+   * Start date and time of the guest configuration assignment compliance status check.
+   */
+  @visibility(Lifecycle.Read)
+  startTime?: utcDateTime;
+
+  /**
+   * End date and time of the guest configuration assignment compliance status check.
+   */
+  @visibility(Lifecycle.Read)
+  endTime?: utcDateTime;
+
+  /**
+   * A value indicating compliance status of the machine for the assigned guest configuration.
+   */
+  @visibility(Lifecycle.Read)
+  complianceStatus?: ComplianceStatus;
+
+  /**
+   * Type of report, Consistency or Initial
+   */
+  @visibility(Lifecycle.Read)
+  operationType?: Type;
+
+  /**
+   * The list of resources for which guest configuration assignment compliance is checked.
+   */
+  @OpenAPI.extension("x-ms-identifiers", #[])
+  resources?: AssignmentReportResource[];
+}
+
+/**
+ * Information about the guest configuration assignment.
+ */
+model AssignmentInfo {
+  /**
+   * Name of the guest configuration assignment.
+   */
+  @visibility(Lifecycle.Read)
+  name?: string;
+
+  /**
+   * Information about the configuration.
+   */
+  configuration?: ConfigurationInfo;
+}
+
+/**
+ * Information about the configuration.
+ */
+model ConfigurationInfo {
+  /**
+   * Name of the configuration.
+   */
+  @visibility(Lifecycle.Read)
+  name?: string;
+
+  /**
+   * Version of the configuration.
+   */
+  @visibility(Lifecycle.Read)
+  version?: string;
+}
+
+/**
+ * Information about the VM.
+ */
+model VMInfo {
+  /**
+   * Azure resource Id of the VM.
+   */
+  @visibility(Lifecycle.Read)
+  id?: string;
+
+  /**
+   * UUID(Universally Unique Identifier) of the VM.
+   */
+  @visibility(Lifecycle.Read)
+  uuid?: string;
+}
+
+/**
+ * The guest configuration assignment resource.
+ */
+model AssignmentReportResource {
+  /**
+   * A value indicating compliance status of the machine for the assigned guest configuration.
+   */
+  @visibility(Lifecycle.Read)
+  complianceStatus?: ComplianceStatus;
+
+  /**
+   * Name of the guest configuration assignment resource setting.
+   */
+  @visibility(Lifecycle.Read)
+  resourceId?: string;
```

Comment:
doc doesn't seem to match property name! is it a name, or a resourceId? Any more specific details?